server/webapp/WEB-INF/rails.new/app/assets/javascripts/vsm_analytics.js

```diff
@@ -88,14 +113,14 @@
     });
   };
 
-  var VSMAnalytics = function VSMAnalytics(data, graphRenderer, showAnalyticsPath, analyticsPanel, analyticsButton) {
+  var VSMAnalytics = function VSMAnalytics(data, graphRenderer, vsmAnalyticsChart, analyticsPanel, analyticsButton) {
```

Comment:
No need to declare a `var` once you've declared a named function.

```
function VSMAnalytics(...) {
  ...
}
```

Will do just fine :)